spec/jwt_spec.rb

```diff
@@ -218,7 +218,7 @@
 
         it 'wrong key should raise JWT::DecodeError' do
           expect do
-            JWT.decode data[alg], wrong_key
+            JWT.decode data[alg], wrong_key, true, algorithm: alg
```

Comment:
Oh, this was a good catch, the test has been wrong all the time, right. Like passing for the wrong reasons.